docs/getting-started/apple/screenshots.md

```diff
@@ -1,8 +1,238 @@
+1. (Objective C only) Add the bridging header to your test class.
+    - `#import "MYUITests-Swift.h"`
+    - The bridging header is named after your test target with -Swift.h appended.
+1. In your UI Test class, click the `Record` button on the bottom left and record your interaction
+1. To take a screenshot, call the following between interactions
+    - Swift: `snapshot("01LoginScreen")`
+    - Objective C: `[Snapshot snapshot:@"01LoginScreen" waitForLoadingIndicator:YES];`
+1. Add the following code to your `setUp()` method:
+
+**Swift**
+
+```swift
+let app = XCUIApplication()
+setupSnapshot(app)
+app.launch()
+```
+
+**Objective C**
+
+```objective-c
+XCUIApplication *app = [[XCUIApplication alloc] init];
+[Snapshot setupSnapshot:app];
+[app launch];
+```
+
+If you have _fastlane_ installed, it's easy to give _snapshot_ a try. First clone the _fastlane_ repo, head over to the _snapshot_ example project, and then run `fastlane snapshot`
+
+```
+$ git clone https://github.com/fastlane/fastlane
+
+$ cd fastlane/snapshot/example
+
+$ fastlane snapshot
+```
+
+![assets/snapshot.gif](assets/snapshot.gif)
+
+# Put Your Screenshots Into Device Frames
+
+_frameit_ helps you beautify your screenshots with devices frames and text by running one simple command. It provides support for:
+
+- Multiple device types
+- Portrait and landscape orientations
+- Black and silver devices
+- Setting a background color and decorating the image with text
+
+![frameit results](assets/frameit-results.png)
+
+## Dependencies
+
+### Installing ImageMagick
+
+_frameit_ depends on a tool called `imagemagick` to do image manipulation. The easiest way to install it is through [homebrew](http://brew.sh/):
+
+```sh
+brew install libpng jpeg imagemagick
+```
+
+### Troubleshooting ImageMagick
+
+If you have installed _imagemagick_ but are seeing error messages like:
+
+```sh
+mogrify: no decode delegate for this image format `PNG'
+```
+
+You may need to reinstall and build from source. Run:
+
+```sh
+brew uninstall imagemagick; brew install libpng jpeg; brew install imagemagick --build-from-source
+```
+
+### Setting Up Device Frames
+
+_fastlane_ uses device frames provided by Apple which need to be downloaded and installed separately for legal reasons. Running `fastlane frameit setup` can walk you through the process.
+
+**TODO: Update this when we start distributing frames ourselves.**
+
+```
+$ fastlane frameit setup
+```
+
+## Running
+
+At its simplest, you can just run:
+
+```sh
+fastlane frameit
+```
+
+A more complex example that configures a lot more about the output can be found in the [fastlane examples](https://github.com/fastlane/examples/tree/master/MindNode/screenshots) project.
+
+### Configure Using `Framefile.json`
+
+Your `Framefile.json` file should be in your `screenshots` folder, as seen in the [example](https://github.com/fastlane/examples/tree/master/MindNode/screenshots), and you can use it to define your configuration:
+
+```json
```

Comment:
I'm not a big fan of spending so much space just for this, as it's really really advanced, most people will never set up frameit with a custom Framefile, maybe we can just link to https://github.com/fastlane/docs/pull/3/files?